jwala-common/src/test/java/com/cerner/jwala/common/rule/jvm/JvmNameRuleTest.java

```diff
@@ -23,7 +23,7 @@ public void testValidNames() {
 
     @Test
     public void testInvalidNames() {
-        final String[] invalidNames = {"", "    ", null};
+        final String[] invalidNames = {"", "    ", null,"***$JVM","JVM123$"};
```

Comment:
Add a test with a / and one with a \ in the name.  Those are file separators and would cause problems if we let those in.